apps/ledger-live-mobile/scripts/e2e-ci.mjs

```diff
@@ -61,6 +62,7 @@ const test_android = async () => {
     --forceExit \\
     --headless \\
     --retries 1 \\
+    --runInBand \\
```

Comment:
Why applying this on Android also? As far as I know we do not have performance issue on Linux runners.

Reply:
interestingly the Android run failed for the first time if I only added it to the ios tests. It also shouldn't cause any issues to apply unilaterally, as far as I understand